mto_route_product_variant/models/product_product.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, models, fields
+
+IS_MTO_HELP = """
+    Check or Uncheck this field if you want this variant to have another
+    mto configration than its template.
+"""
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_mto = fields.Boolean(
+        string="Variant is Mto",
+        compute="_compute_is_mto",
+        inverse="_inverse_is_mto",
+        store="true",
+        help=IS_MTO_HELP,
+        index=True
```

Comment:
What's the reason for this? An index on a boolean column will almost never be used...